AsyncDisplayKit/ASCollectionView.h

```diff
@@ -205,17 +205,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setTuningParameters:(ASRangeTuningParameters)tuningParameters forRangeMode:(ASLayoutRangeMode)rangeMode rangeType:(ASLayoutRangeType)rangeType ASDISPLAYNODE_DEPRECATED_MSG("Use ASCollectionNode method instead.");
 
-/**
- * Scrolls the collection to the given item.
- *
- * @param indexPath The index path of the item.
- * @param scrollPosition Where the row should end up after the scroll.
- * @param animated Whether the scroll should be animated or not.
- */
+- (nullable __kindof UICollectionViewCell *)cellForItemAtIndexPath:(NSIndexPath *)indexPath ASDISPLAYNODE_DEPRECATED_MSG("Use ASCollectionNode method instead.");
```

Comment:
What does `__kindof` indicate here? I understand that it indicates subclasses are valid for collection generics, but subclasses are already valid as return values? Does it shut-up the compiler when assign to a subclass?